test/TensorFlowNET.UnitTest/ExamplesTests/ExamplesTest.cs

```diff
@@ -51,6 +51,7 @@ public void LinearRegression()
             new LinearRegression() { Enabled = true }.Run();
         }
 
+        [Ignore("not working")]

```

Comment:
Why do you ingore this?

![image](https://user-images.githubusercontent.com/1705364/61947996-475b3800-af6c-11e9-919a-e5c3b4e3983c.png)

Reply:
those tests are all failing on my machine.

Reply:
I've updated visual studio and they are working now. not sure why. maybe an issue with a file handle 🤔

updated the change request.